CONTRIBUTING.md

```diff
@@ -9,74 +10,96 @@ Issues reported on [GitHub](https://github.com/themoeway/yomitan/issues) should
-- `--version <version>` - Sets the version number in the extension manifest. Defaults to 0.0.0.0 if not set.
+- `--manifest <target>` - Overwrites `ext/manifest.json` with the manifest
+  variant for the specified build target.
+- `--dryRun` - Runs the full build process (excluding zip building), checking
+  that the configuration is valid.
+- `--dryRunBuildZip` - If `--dryRun` is also specified, zip building will also
+  be performed in memory; no files are created.
+- `--version <version>` - Sets the version number in the extension manifest.
+  Defaults to 0.0.0.0 if not set.
 
 If no arguments are specified, the command is equivalent to `build.bat --all`.
 
 ### Build Tools
 
-The build process can use the [7-zip](https://www.7-zip.org/) archiving tool to create the packed zip builds
-if the 7-zip executable (either `7z` or `7za`) is found in the `PATH` environment variable.
-Otherwise, the [JSZip](https://stuk.github.io/jszip/) API is used to generate the files.
-7-zip typically provides better compression than JSZip, but the files are otherwise equivalent.
+The build process can use the [7-zip](https://www.7-zip.org/) archiving tool to
+create the packed zip builds if the 7-zip executable (either `7z` or `7za`) is
+found in the `PATH` environment variable. Otherwise, the
+[JSZip](https://stuk.github.io/jszip/) API is used to generate the files. 7-zip
+typically provides better compression than JSZip, but the files are otherwise
+equivalent.
 
 ## Manifest
 
-Manifest variants for different build targets are specified in [manifest-variants.json](dev/data/manifest-variants.json).
-This file is used to generate the `ext/manifest.json` file included in the extension.
-The generated `ext/manfiest.json` should not be committed.
+Manifest variants for different build targets are specified in
+[manifest-variants.json](dev/data/manifest-variants.json). This file is used to
+generate the `ext/manifest.json` file included in the extension. The generated
+`ext/manfiest.json` should not be committed.
 
 ## Style
 
-Linting rules are defined for a few types of files, and validation is performed as part of the standard tests
-run by `npm test` and the continuous integration process.
+Linting rules are defined for a few types of files, and validation is performed
+as part of the standard tests run by `npm test` and the continuous integration
+process.
 
 - [.eslintrc.json](.eslintrc.json) rules are used for JavaScript files.
 - [.stylelintrc.json](.stylelintrc.json) rules are used for CSS files.
 - [.htmlvalidate.json](.htmlvalidate.json) rules are used for HTML files.
 
-In addition, the [Markdown All in One VSCode extension](https://github.com/yzhang-gh/vscode-markdown) is used for formatting markdown files and automatically updating the table of contents.
+In addition, the
+[Markdown All in One VSCode extension](https://github.com/yzhang-gh/vscode-markdown)
```

Comment:
Since we can have formatting validation, we can remove this section. In addition, this extension is specific to VSCode and it seems like yomitan developers wasn't using it anyway before #420 .